common/src/main/java/org/apache/uniffle/common/segment/FixedSizeSegmentSplitter.java

```diff
@@ -84,11 +83,10 @@ private static List<ShuffleDataSegment> transIndexDataToSegments(
         if (dataFileLen != -1 && totalLength > dataFileLen) {
           LOGGER.info(
               "Abort inconsistent data, the data length: {}(bytes) recorded in index file is greater than "
-                  + "the real data file length: {}(bytes). Partition id: {}. "
+                  + "the real data file length: {}(bytes). "
                   + "This may happen when the data is flushing, please ignore.",
               totalLength,
-              dataFileLen,
-              BlockId.getPartitionId(blockId));
```

Comment:
Why do we remove partitionId here?

Reply:
The actual block id layout is not available here, and wiring it through from the client would mean to add it to many method signature calls. All this only for logging.

At least we should log the `long` block id here so it can be de-cyphered manually if needed.

Reply:
ok.